### DISCUSSION TITLE
Welcome to Discord-Auto-Advertiser Discussions!

### DISCUSSION THREAD

**Discussion by xathail** [xzerios/Discord-Auto-Advertiser#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JordanP244656** [comment]
Is there a discord that I can get help with this in?

**Reply by xathail** [reply]
If by that you mean a server, then no. What's your Discord username?

Edit: I'd much rather you actually try to figure out how to run it yourself and open an issue if there's something unclear that you run into.

**Reply by JordanP244656** [reply]
So should I send my username or no?

**Reply by xathail** [reply]
Yeah.
